test/helpers/helpers.js

```diff
@@ -1,36 +1,35 @@
+const Promise = require('bluebird')
+
 const assertRevert = async (receiptPromise, reason) => {
-  try {
-    await receiptPromise
-  } catch (error) {
-    if (reason) {
-      assert.include(error.message, reason, 'Incorrect revert reason')
+    try {
+        await receiptPromise
+    } catch (error) {
+        if (reason) {
+            assert.include(error.message, reason, 'Incorrect revert reason')
+        }
+        return
     }
-    return
-  }
-  assert.fail(`Expected a revert for reason: ${reason}`)
+    assert.fail(`Expected a revert for reason: ${reason}`)
 }
 
 const getLog = (receipt, logName, argName) => {
-  const log = receipt.logs.find(({ event }) => event === logName)
-  return log ? log.args[argName] : null
+    const log = receipt.logs.find(({event}) => event === logName)
+    return log ? log.args[argName] : null
 }
 
 const deployedContract = receipt => getLog(receipt, 'NewAppProxy', 'proxy')
 
-const getSignatureFields = signature => {
-  signature = signature.slice(2) // remove 0x
-  const v = web3.utils.toDecimal(`0x${signature.slice(128, 130)}`)
-
-  return {
-    v: v != 27 && v != 28 ? v + 27 : v,
-    r: `0x${signature.slice(0, 64)}`,
-    s: `0x${signature.slice(64, 128)}`,
-  }
-}
+const advanceTime = web3 => Promise.promisify(function(delay, done) {
```

Comment:
There's an `@aragon/test-helpers` function `timeTravel` which is used for this as well. 
Maybe we can use that one instead so we don't add another dependency?

Reply:
Have you used it/can you point me to any files that do? I don't think I could get it to work.

Reply:
Copied their implementation and fixed for web3js 1.0.0. The @aragon/test-helpers implementation uses `sendAsync` which got deprecated in favour of `send`. Also removed the dependency.